erpnext/projects/doctype/task/task.py

```diff
@@ -99,7 +99,14 @@ def update_time_and_costing(self):
 
 	def update_project(self):
 		if self.project and not self.flags.from_project:
-			frappe.get_doc("Project", self.project).update_project()
+			project = frappe.get_doc("Project", self.project)
+			for task in project.tasks:
+				if task.task_id == self.name:
+					task.start_date = self.exp_start_date
```

Comment:
This should be handled here

https://github.com/frappe/erpnext/blob/3866e132f71dfe98cb2dc95bf613a2b1d5de1cb7/erpnext/projects/doctype/project/project.py#L142

Reply:
but it synced when task is updated so put it into task.py

Reply:
it sync the tasks if something is updated from project side.
But Here is the opposite scenario when we update project the changes go to tasks.
But when we update the task(date) it does not go to the project.

Reply:
done in different way

Reply:
Now fixing travis